src/styles/variables.scss

```
@@ -11,17 +11,19 @@ $grey-3: #dbdbdb;
 
 $grey-4: #a6a6a6;
 
-$gray-light: #969696 !default;
-$color-gray: #818181;
+// Interim color variable for migration purposes, as the step between `$grey-4`
+// and `$grey-5` is a doozy. Represents `base-semi` in proposed future palette,
```

```suggestion
// and `$grey-5` is quite large. Represents `base-semi` in proposed future palette,
```

It wouldn't have been immediately obvious to me what this meant if I hadn't read the PR description or looked at the color values carefully.